wiki/Ranking_Criteria/en.md

```diff
@@ -177,7 +178,7 @@ This category contains explicit allowance statements of concepts and rules that
 - **[Uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) must be used to accurately beatmap the song's time signatures.** If an incorrect time signature lasts for more than one bar, an uninherited timing point must be added on the next downbeat to reset the time signature. For #/4-signatures unsupported by the editor, metronome resets or editing of the `.osu` file are acceptable. For other unsupported time signatures, refer to this [exemplary chart](/wiki/shared/timing/Timing_signature_reference_chart.png), and see the [Timing songs with #/8-signatures](/wiki/Guides/Timing_Songs_With_8-Signatures) guide for further information.
 - **Beatmaps must be perfectly timed.** This means [BPM](/wiki/Beatmapping/Beats_per_minute) and [offset](/wiki/Beatmapping/Offset) are exactly synchronised with the song. Beatmaps with constantly changing BPM may be impossible to perfectly time and should instead be as accurate as possible without negatively affecting gameplay. Complex timing during breaks or spinners is optional.
 - **[Uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) must be the same in every difficulty of a beatmap.** Each point must have the same [BPM](/wiki/Beatmapping/Beats_per_minute) and [offset](/wiki/Beatmapping/Offset) in each difficulty.
-- **There must not be extra [uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) in any difficulty.** These may accidentally affect main-menu pulsing, add unwanted sounds to the [Nightcore mod](/wiki/Game_modifier/Nightcore), or cause timing to shift. Acceptable uses include:
+- **There must not be extra [uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) in any difficulty.** These may accidentally affect main-menu pulsing, add unwanted sounds to the [Nightcore mod](/wiki/Game_modifier/Nightcore), or cause timing to shift. Exceptions to this rule include:
```

Comment:
hmm the original wording led to misinterpretations already, so i'd rather not keep that, but i'm open to other ideas. "Acceptable uses of extra uninherited timing points include:" is the intent, so maybe that? it's a mouthful though

Reply:
How about rewording the whole paragraph to read more like

"Uninherited timing points should only be used when required for timing purposes..."

Also I'm not sure uninherited is a word.

Reply:
Inherited and uninherited are terms mappers have used for ages

I suppose it would be:

```suggestion
- **[Uninherited timing points](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point) should only be used when required for timing purposes.** Unnecessary timing points may accidentally affect main-menu pulsing, add unwanted sounds to the [Nightcore mod](/wiki/Game_modifier/Nightcore), or cause timing to shift. Other acceptable uses include:
```

Reply:
Two things:

- The listed acceptable uses are all timing related, so I'd drop the "Other"
- Maybe change "accidentally" to "unintentionally"